src/js/components/Meter/Circle.js

```diff
@@ -145,12 +145,14 @@ const Circle = forwardRef((props, ref) => {
     viewBoxHeight = width / 2;
   } else if (size === 'full') {
     viewBoxHeight = '100%';
+  } else if (size === 'full' && type === 'semicircle') {
+    viewBoxHeight = '100%';
```

Comment:
You'll never hit this because of 144-146.
It's also weird because this is called viewBoxHeight but it's not used for the viewBox but is used for height.
And for semicircle don't you want the view box to have a height of width/2 even for `full`?

How about something like:
```jsx
const viewBoxHeight = (type === 'semicircle') ? width / 2 : width;

return (
<StyledMeter
  ref={ref}
  viewBox={`0 0 ${width} ${viewBoxHeight}`}
  width={size === 'full' ? '100%' : width}
  height={size === 'full' ? '100%' : viewBoxHeight}
  {...rest}
>
  {track}
  {paths}
  {pathCaps}
</StyledMeter>
```

Reply:
we would also want to check for `full` correct ? to then use '100%' 
otherwise is they pass `size="full"` then the meter will look smaller than `medium`

Reply:
I agree! I wasn't entirely sure if the `semicircle` even when full should be `width/2`